src/main/java/com/jcabi/xml/ClasspathInput.java

```diff
@@ -121,10 +121,12 @@ public String getEncoding() {
     @Override
     public String getStringData() {
         try {
-            return IOUtils.toString(
+            final String data = IOUtils.toString(
                 this.stream,
                 Charset.forName("UTF-8")
             );
+            this.stream.close();
```

Comment:
@amihaiemil we  could ask him
@yegor256 could you confirm that `ClasspathInput` should close the stream by itself?